packages/lib/response/service.ts

```diff
@@ -351,8 +396,12 @@ export const updateResponse = async (
       tags: responsePrisma.tags.map((tagPrisma: { tag: TTag }) => tagPrisma.tag),
     };
 
+    revalidateTag(responseCacheTag.single(response.id));
```

Comment:
shouldn't here be an easy way to say: the response changes (update, delete, create), please revalidate all caches connected to this response. ?
e.g. `responseCache.single(response)`
Also not sure about this...

Reply:
I guess this is what you sent me as well on slack, agree.